katas/langtonant/README.md

```diff
@@ -3,7 +3,7 @@
 The purpose of this kata is to implement a program that allow us to represent a Langton's Ant behavior. Langton's ant is a two-dimensional universal Turing machine with a very simple set of rules but complex emergent behavior, read mode abour it [here](https://en.wikipedia.org/wiki/Langton%27s_ant).
 
 ## Rules
-The Langnton's Ant consists of a group of squares on a plane (grid) colored variously either black or white. We arbitrarily identify one square as the “ant”. The ant can travel in any of the four cardinal directions at each step it takes. The “ant” moves according to the rules below:
+The Langnton's Ant consists of a group of squares on a plane (grid) colored variously either black or white. We arbitrarily identify one square as the “ant”. The ant can travel in any of the four cardinal orientations at each step it takes. The “ant” moves according to the rules below:
```

Comment:
ℹ️ this change will generated furutre conflicts for future merges, i recommend to change for old announcement,. thas all
-✅ The lint task hasnt be problems
-✅ The package conflict has been resolved
-✅  The Kata's code looks funtional